etl_export/src/main/java/ovirt_engine_dwh/aggregationtodaily_4_5/AggregationToDaily.java

```diff
@@ -18252,57 +18076,39 @@ public boolean equals(Object obj) {
 								row20.max_write_latency_seconds);
 					}
 
-					if (row20.read_ops_per_second == null) {
-						pstmt_tJDBCOutput_13
-								.setNull(15, java.sql.Types.INTEGER);
+					if (row20.read_ops_total_count == null) {
+						pstmt_tJDBCOutput_13.setNull(15, java.sql.Types.DOUBLE);
 					} else {
-						pstmt_tJDBCOutput_13.setLong(15,
-								row20.read_ops_per_second);
+						pstmt_tJDBCOutput_13.setDouble(15,
+								row20.read_ops_total_count);
 					}
 
-					if (row20.max_read_ops_per_second == null) {
-						pstmt_tJDBCOutput_13
-								.setNull(16, java.sql.Types.INTEGER);
-					} else {
-						pstmt_tJDBCOutput_13.setLong(16,
-								row20.max_read_ops_per_second);
-					}
-
-					if (row20.write_ops_per_second == null) {
-						pstmt_tJDBCOutput_13
-								.setNull(17, java.sql.Types.INTEGER);
-					} else {
-						pstmt_tJDBCOutput_13.setLong(17,
-								row20.write_ops_per_second);
-					}
-
-					if (row20.max_write_ops_per_second == null) {
-						pstmt_tJDBCOutput_13
-								.setNull(18, java.sql.Types.INTEGER);
+					if (row20.write_ops_total_count == null) {
+						pstmt_tJDBCOutput_13.setNull(16, java.sql.Types.DOUBLE);
 					} else {
-						pstmt_tJDBCOutput_13.setLong(18,
-								row20.max_write_ops_per_second);
+						pstmt_tJDBCOutput_13.setDouble(16,
+								row20.write_ops_total_count);
 					}
 
 					if (row20.flush_latency_seconds == null) {
-						pstmt_tJDBCOutput_13.setNull(19, java.sql.Types.DOUBLE);
+						pstmt_tJDBCOutput_13.setNull(17, java.sql.Types.DOUBLE);
 					} else {
-						pstmt_tJDBCOutput_13.setDouble(19,
+						pstmt_tJDBCOutput_13.setDouble(17,
 								row20.flush_latency_seconds);
 					}
 
 					if (row20.max_flush_latency_seconds == null) {
-						pstmt_tJDBCOutput_13.setNull(20, java.sql.Types.DOUBLE);
+						pstmt_tJDBCOutput_13.setNull(18, java.sql.Types.DOUBLE);
 					} else {
-						pstmt_tJDBCOutput_13.setDouble(20,
+						pstmt_tJDBCOutput_13.setDouble(18,
 								row20.max_flush_latency_seconds);
 					}
 
 					if (row20.vm_disk_configuration_version == null) {
 						pstmt_tJDBCOutput_13
-								.setNull(21, java.sql.Types.INTEGER);
```

Comment:
@avlitman why are there changes here?

Reply:
This is the position of the columns, since we deleted the two max_read/write_ops_per_second columns, the numbers dropped by 2